sample/app/src/main/res/drawable/test_drawable.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#24CF4D"
```

Comment:
@simonschiller I've added this drawable to extend the test in [ReleaseReportTest.kt].
But CI complains about Unused resources, what do you suggest I can go ahead and use this drawable in the MainActivity layout or should we remove it?

Reply:
Should be fine to use it in the layout I think.